app/scripts/modules/amazon/src/serverGroup/details/resize/AmazonResizeServerGroupModal.tsx

```diff
@@ -0,0 +1,424 @@
+                  </a>
+                  .
+                </span>
+              )}
+              {advancedMode && (
+                <span>
+                  Scaling policies will not take effect when <b>Min</b> is the same as <b>Max</b>.
+                </span>
+              )}
+            </div>
+          </div>
+        </div>
+      );
+    }
+    if (this.isDesiredControlledByAutoscaling()) {
+      return (
+        <div className="form-group">
+          <div className="col-md-8 col-md-offset-1">
+            <div className="well-compact alert alert-warning">
+              <p>
+                <b>Desired</b> capacity is managed by Autoscaling Policies.
+              </p>
+              <p>
+                If you need to scale <b>down</b> this server group, set <b>Max</b> to the new desired size.
+              </p>
+              <p>
+                If you need to scale <b>up</b> this server group, set <b>Min</b> to the new desired size.
+              </p>
+            </div>
+          </div>
+        </div>
+      );
+    }
+    return null;
+  }
+
+  public render() {
+    const { serverGroup } = this.props;
+    const { advancedMode, initialValues, platformHealthOnlyShowOverride } = this.state;
+    const { TaskMonitorWrapper } = NgReact;
+    return (
+      <>
+        <TaskMonitorWrapper monitor={this.state.taskMonitor} />
+        <Formik<IAmazonResizeServerGroupValues>
+          ref={this.formikRef}
+          initialValues={initialValues}
+          validate={this.validate}
+          isInitialValid={true}
+          onSubmit={this.submit}
+          render={formik => {
+            return (
+              <>
+                <Form className="form-horizontal">
+                  <Modal.Header>
+                    <h3>Resize {serverGroup.name}</h3>
+                  </Modal.Header>
+                  <ModalClose dismiss={this.close} />
+                  <Modal.Body>
+                    {advancedMode && this.renderAdvancedMode()}
+                    {!advancedMode && this.renderSimpleMode()}
+                    {this.renderScalingPolicyWarning(formik)}
+                    {this.renderCapacityConstraintSelector()}
+                    {platformHealthOnlyShowOverride && (
+                      <div className="form-group">
+                        <div className="col-md-8 col-md-offset-3">
+                          <PlatformHealthOverride
+                            command={this.state}
+                            platformHealthType="Amazon"
+                            onChange={this.platformHealthOverrideChanged}
+                            showHelpDetails={true}
+                          />
+                        </div>
+                      </div>
+                    )}
+                    <TaskReason reason={formik.values.reason} onChange={val => formik.setFieldValue('reason', val)} />
+                  </Modal.Body>
+                </Form>
+                <AwsModalFooter
```

Comment:
can this component get moved up a line (inside the `Form`) and then remove the `<>` React.Partial tags?

Reply:
You would think! And I would like to! But it ends up breaking a bunch of CSS, so part of the footer is cut off.